### DISCUSSION TITLE
Dúvidas sobre a ferramenta

### DISCUSSION THREAD

**Discussion by maxmoura03** [amidaware/tacticalrmm#1891] [General]
Não conhecia a ferramenta até poucos dias atrás, gostaria que alguém pudesse me tirar algumas dúvidas...

As máquinas que possuem o agente do Tactical RMM estão seguras contra acessos indevidos, uma pessoa não poderia pegar o controle do computador mesmo sem o acesso do painel ou usar de alguma forma para realizar um ataque ransomware?

Minha dúvida é se é seguro implementar e usar em diversos computadores, ficando vulnerável a acessos indevidos, capturas de senha, comprometimento de rede e etc...

Estou perguntando por realmente ser leigo na ferramenta

**Comment by ssteeltm** [comment]
Ola amigo, com certeza é uma ferramenta segura. Utilizado e Validado por varios usuarios no mundo todo.
A ferramenta de acesso remoto, embutida se chama MeshCentral, esta pode ser personalizada de varias formas para atingir a forma que deseja trabalhar, sendo intrusivo ou nao ao acessar na maquina. ( por exemplo, avisar ao usuario que alguem entrou remoto).

Sugiro uma leitura completa da documentação: https://docs.tacticalrmm.com/

E tambem tem o discord, aonde todos podemos conversar em tempo real, sobre problemas, sugestoes e duvidas.
https://discord.gg/upGTkWp

**Comment by silversword411** [comment]
English only please.
